wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/player/NowPlayingScreen.kt

```diff
@@ -2,20 +2,24 @@ package au.com.shiftyjelly.pocketcasts.wear.ui.player
 
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
 import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.wear.compose.material.MaterialTheme
 import androidx.wear.compose.material.Scaffold
-import au.com.shiftyjelly.pocketcasts.wear.ui.component.AnimatedPlayerScreenMediaDisplay
-import com.google.android.horologist.audio.ui.ExperimentalHorologistAudioUiApi
+import androidx.wear.compose.material.Text
+import au.com.shiftyjelly.pocketcasts.R
+import au.com.shiftyjelly.pocketcasts.localization.helper.TimeHelper
+import au.com.shiftyjelly.pocketcasts.wear.ui.component.MarqueeTextMediaDisplay
 import com.google.android.horologist.media.ui.components.PodcastControlButtons
+import com.google.android.horologist.media.ui.components.display.MessageMediaDisplay
 import com.google.android.horologist.media.ui.screens.player.PlayerScreen
-import com.google.android.horologist.media.ui.state.PlayerUiController
-import com.google.android.horologist.media.ui.state.PlayerUiState
 
 object NowPlayingScreen {
-    const val argument = "playableUuid"
-    const val route = "playable/{$argument}"
-    fun navigateRoute(playableUuid: String) = "playable/$playableUuid"
+    const val route = "now_playing"
```

Comment:
Instead of having an optional argument that would start playing an episode as part of the navigation, I'm instead having the screen/view-model where the new episode is selected directly trigger playback via the `PlaybackManager`. That way navigation is only concerned with navigation and not also handling the possibility of starting playback of a new episode.